test/cookbooks/test/recipes/contrib.rb

```diff
@@ -2,6 +2,6 @@
 apt_update 'update' if platform_family?('debian')
 
 node.default['postgresql']['password']['postgres'] = 'iloverandompasswordsbutthiswilldo'
-node.default['postgresql']['contrib']['extensions'] = %w(plpgsql xml2)
+node.default['postgresql']['contrib']['extensions'] = %w(plpgsql xml2) unless platform_family?('rhel') && Gem::Version.new(node['platform_version']) < Gem::Version.new('7')
```

Comment:
It's probably simpler here to just do node['platform_version'].to_i instead of using Gem::Version